packages/common/src/core/types/IHubSite.ts

```diff
@@ -7,12 +7,30 @@ import { IWithCatalog, IWithCatalogs } from "../traits/IWithCatalog";
 import { IWithLayout } from "../traits/IWithLayout";
 import { IWithPermissions } from "../traits/IWithPermissions";
 
+export interface IHubSiteUrlInfo {
```

Comment:
maybe add a comment that while not used for any other entity (thus why it's in this file) this is extracted into an interface to ensure IHubSiteEditor shares a consistent structure (this took a me little 🤔)